### ISSUE TITLE
Copy Site plus maybe a bug

### ISSUE THREAD

**Issue by MCNorthwood** [PiranhaCMS/piranha.core#418]
Is there a way of copying all the pages (including the blog with posts) for a new site so it would mean there's a mirror image of the site. To then be able to make minor tweaks without having to change the current default site.
That would be a feature request, more than likely.

The bug I saw also was in the media section, whenever you click on an image and the a pdf it shows the previous image you clicked in the modal that pops up.

**Comment by tidyui** [comment]
Good call! Would it be possible to edit this issue a bit and split it into two separate issues? Makes it a lot easier to label, assign and prioritize!

Best regards

Håkan

**Comment by MCNorthwood** [comment]
Yeah alright!
